configdialog/lxqtpageselectwidget.cpp

```diff
@@ -102,12 +103,14 @@ QSize PageSelectWidget::sizeHint() const
     int w = 0;
     for(int i=0; i< count(); ++i)
     {
-        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignLeft | Qt::TextWordWrap, item(i)->text());
+        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignHCenter | Qt::TextWordWrap, item(i)->text());
         w = qMax(w, rect.width());
     }
+    // consider the icon size too
+    w = qMax(w, QApplication::style()->pixelMetric(QStyle::PM_IconViewIconSize));
```

Comment:
> I was asking about...

Theoretically there shouldn't be any difference because there's no hard-coded value here. I just don't trust hard-coded styles because of my experience from Kvantum: there are so many good devs who make strange mistakes when it comes to QStyle.